pkg/awsclient/client.go

```diff
@@ -339,3 +345,14 @@ func NewClientFromSecret(secret *corev1.Secret, region string) (Client, error) {
 		tagClient:     resourcegroupstaggingapi.New(s),
 	}, nil
 }
+
+func awsChinaEndpointResolver(service, region string, optFns ...func(*endpoints.Options)) (endpoints.ResolvedEndpoint, error) {
+	if service != route53.EndpointsID || region != constants.AWSChinaRoute53Region {
+		return endpoints.DefaultResolver().EndpointFor(service, region, optFns...)
+	}
+
+	return endpoints.ResolvedEndpoint{
+		URL:         "https://api.route53.cn",
```

Comment:
Should this be `https://route53.amazonaws.com.cn`?

Reply:
@staebler Yes, we should use https://route53.amazonaws.com.cn, this is from AWS SA:

```
For the China Route53, there are 2 endpoints,  see below

    1  api.route53.cn

    2 route53.amazonaws.com.cn

Based on the info from service team , both 2 endpoints are workable after GA. But the new endpoint(the 2nd ) is recommended.
```

Reply:
Waiting on this change but otherwise LGTM.

Reply:
I made this change and verified that everything is still working.